content/en/includes/task-tutorial-prereqs.md

```diff
@@ -2,7 +2,6 @@ You need to have a Kubernetes cluster, and the kubectl command-line tool must
 be configured to communicate with your cluster. It is recommended to run this tutorial on a cluster with at least two nodes that are not acting as control plane hosts. If you do not already have a
 cluster, you can create one by using
 [minikube](https://minikube.sigs.k8s.io/docs/tutorials/multi_node/)
-or you can use one of these Kubernetes playgrounds:
+or you can use this Kubernetes playground:
```

Comment:
Let's keep the original line 5 and replace the Katacoda link to Killercoda
* [Killercoda](https://killercoda.com/playgrounds/scenario/kubernetes)